phys2bids/phys2bids.py

```diff
@@ -340,18 +402,25 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
         bids.dataset_description_file(outdir)
         # Generate README file if it doesn't exist already.
         bids.readme_file(outdir)
-        cp(heur_file, os.path.join(conversion_path,
-           os.path.splitext(os.path.basename(heur_file))[0] + '.py'))
+        cp(
+            heur_file,
+            os.path.join(
+                conversion_path,
+                os.path.splitext(os.path.basename(heur_file))[0] + ".py",
+            ),
+        )
     elif heur_file is not None and sub is None:
-        LGR.warning('While "-heur" was specified, option "-sub" was not.\n'
-                    'Skipping BIDS formatting.')
+        LGR.warning(
+            'While "-heur" was specified, option "-sub" was not.\n'
```

Comment:
Odd

Reply:
Do you mean the new line is odd, or the fact that this line uses single quotes and the next uses double quotes?